WordPress/Classes/ViewRelated/What's New/WhatIsNewViewController.swift

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
Please consider documenting this class, it would help me to understand what's going on :)